cli/command/trust/signer_add.go

```diff
@@ -44,9 +46,13 @@ func newSignerAddCommand(dockerCli command.Cli) *cobra.Command {
 	return cmd
 }
 
+var validSignerName = regexp.MustCompile(`^[a-z0-9]+[a-z0-9\_\-]*$`).MatchString
+
 func addSigner(cli command.Cli, options *signerAddOptions) error {
 	signerName := options.signer
-
+	if !validSignerName(signerName) {
+		return fmt.Errorf("signer name \"%s\" must not contain special characters", signerName)
```

Comment:
can this error message be rephrased to "...must not contain uppercase or special characters"?